app/domain/authentication/authn_jwt/consts.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Authentication
+  module AuthnJwt
```

Comment:
please freeze all the strings in this class

Reply:
Freezing string itself not helps . We need to freeze the variable.
But to freeze the variable we will need to add each line for each variable and its bad readability.

Reply:
ok np